src/test/java/org/folio/rest/api/StorageTestSuite.java

```diff
@@ -40,6 +42,7 @@
 
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
+  CallNumberUtilsTest.class,
```

Comment:
Why is this test class included in the suite when it doesn't need infrastructure?